packages/main/cypress/specs/BusyIndicator.cy.tsx

```diff
@@ -24,3 +29,186 @@ describe("Rendering", () => {
 			.should("exist");
 	});
 });
+
+describe("BusyIndicator general interaction", () => {
+	it("tests event propagation", () => {
+		const onClickStub = cy.stub().as("myStub");;
+		cy.mount(
+			<BusyIndicator id="busy-tree" class="busyindicator8auto">
+				<Tree id="treeDynamic">
+					<TreeItem id="treeItem" text="Has preloaded children" onClick={onClickStub}>
+						<TreeItem text="Child 1"></TreeItem>
+						<TreeItem text="Child 2"></TreeItem>
+					</TreeItem>
+				</Tree>
+			</BusyIndicator>
+		);
+
+		cy.get("#treeItem")
+			.shadow()
+			.find("ui5-icon.ui5-li-tree-toggle-icon")
+			.realClick();
+
+		cy.realPress("Space");
+
+		cy.get("@myStub").should("not.have.been.called");
+	});
+
+	it("test activation", () => {
+		cy.mount(
+			<BusyIndicator id="busy-container" class="busyindicator4auto" size="M" text="Loading">
+				<List id="fetch-list" no-data-text="No Data" header-text="Available Items" class="busyindicator5auto"></List>
+			</BusyIndicator>
```

Comment:
Replace the list with simple div or span. Remove attributes `class`, `size` and `text`